samples/wifi/shell/sample.yaml

```diff
@@ -23,3 +23,17 @@ tests:
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf7002dk_nrf5340_cpuapp
     tags: ci_build
+  sample.nrf7002.shell.scan_only_7002:
+    build_only: true
+    extra_args: OVERLAY_CONFIG=overlay-scan-only.conf
+    integration_platforms:
+      - nrf7002dk_nrf5340_cpuapp
+    platform_allow: nrf7002dk_nrf5340_cpuapp
+    tags: ci_build
+  sample.nrf7002.shell.scan_only_9160:
+    build_only: true
+    extra_args: OVERLAY_CONFIG=overlay-scan-only.conf SHIELD=nrf7002_ek
```

Comment:
Please, add instructions to Readme how to compile by using west and use with nrf9160